<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>HTTP/FTP 0.3</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >:/resources/images/pluginicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="tabPosition" >
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="ActiveJobsTab_" >
       <attribute name="title" >
        <string>Active</string>
       </attribute>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="ContextableList" name="TasksList_" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="textElideMode" >
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <property name="uniformRowHeights" >
           <bool>true</bool>
          </property>
          <property name="itemsExpandable" >
           <bool>false</bool>
          </property>
          <column>
           <property name="text" >
            <string>State</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Local name</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Progress</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Speed</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Download time</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>ETA</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Downloaded</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Total</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="FinishedJobsTab_" >
       <attribute name="title" >
        <string>Finished</string>
       </attribute>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="ContextableList" name="FinishedList_" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0" >
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="textElideMode" >
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <property name="uniformRowHeights" >
           <bool>true</bool>
          </property>
          <property name="itemsExpandable" >
           <bool>false</bool>
          </property>
          <column>
           <property name="text" >
            <string>Local name</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Average speed</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Time to complete</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuJobs_" >
    <property name="title" >
     <string>Jobs</string>
    </property>
    <widget class="QMenu" name="menuActive" >
     <property name="title" >
      <string>Active</string>
     </property>
     <addaction name="ActionStart_" />
     <addaction name="ActionStop_" />
     <addaction name="ActionStartAll_" />
     <addaction name="ActionStopAll_" />
     <addaction name="ActionGetFileSize_" />
     <addaction name="ActionSchedule_" />
     <addaction name="ActionJobProperties_" />
    </widget>
    <widget class="QMenu" name="menuFinished" >
     <property name="title" >
      <string>Finished</string>
     </property>
     <addaction name="ActionRemoveFinished_" />
    </widget>
    <addaction name="ActionAddJob_" />
    <addaction name="ActionRemoveJob_" />
    <addaction name="separator" />
    <addaction name="menuActive" />
    <addaction name="menuFinished" />
   </widget>
   <widget class="QMenu" name="MenuTools_" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="ActionAutoadjust_" />
    <addaction name="ActionActiveColumns_" />
    <addaction name="ActionFinishedColumns_" />
    <addaction name="separator" />
    <addaction name="ActionPreferences_" />
   </widget>
   <addaction name="MenuJobs_" />
   <addaction name="MenuTools_" />
  </widget>
  <widget class="QToolBar" name="Active_" >
   <property name="windowTitle" >
    <string>Active jobs</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionAddJob_" />
   <addaction name="ActionRemoveJob_" />
   <addaction name="separator" />
   <addaction name="ActionStart_" />
   <addaction name="ActionStop_" />
   <addaction name="ActionStartAll_" />
   <addaction name="ActionStopAll_" />
   <addaction name="ActionGetFileSize_" />
   <addaction name="ActionSchedule_" />
   <addaction name="ActionJobProperties_" />
  </widget>
  <widget class="QToolBar" name="MainToolbar_" >
   <property name="windowTitle" >
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionAutoadjust_" />
   <addaction name="ActionActiveColumns_" />
   <addaction name="ActionFinishedColumns_" />
   <addaction name="separator" />
   <addaction name="ActionPreferences_" />
  </widget>
  <widget class="QToolBar" name="Finished_" >
   <property name="windowTitle" >
    <string>Finished jobs</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionRemoveFinished_" />
  </widget>
  <widget class="QStatusBar" name="statusBar" >
   <property name="sizeGripEnabled" >
    <bool>true</bool>
   </property>
  </widget>
  <action name="ActionAddJob_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/addjob.png</iconset>
   </property>
   <property name="text" >
    <string>Add job...</string>
   </property>
  </action>
  <action name="ActionRemoveJob_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/deletejob.png</iconset>
   </property>
   <property name="text" >
    <string>Remove job</string>
   </property>
  </action>
  <action name="ActionStart_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/startjob.png</iconset>
   </property>
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="ActionStop_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/stopjob.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="ActionStartAll_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/startall.png</iconset>
   </property>
   <property name="text" >
    <string>Start all</string>
   </property>
  </action>
  <action name="ActionStopAll_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/stopall.png</iconset>
   </property>
   <property name="text" >
    <string>Stop all</string>
   </property>
  </action>
  <action name="ActionGetFileSize_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/getfilesize.png</iconset>
   </property>
   <property name="text" >
    <string>Get file size</string>
   </property>
  </action>
  <action name="ActionSchedule_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/schedule.png</iconset>
   </property>
   <property name="text" >
    <string>Schedule</string>
   </property>
  </action>
  <action name="ActionJobProperties_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/properties.png</iconset>
   </property>
   <property name="text" >
    <string>Job properties...</string>
   </property>
  </action>
  <action name="ActionRemoveFinished_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/deletejob.png</iconset>
   </property>
   <property name="text" >
    <string>Remove finished</string>
   </property>
  </action>
  <action name="ActionAutoadjust_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/autoadjustiface.png</iconset>
   </property>
   <property name="text" >
    <string>Autoadjust interface</string>
   </property>
  </action>
  <action name="ActionActiveColumns_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/selectcolumns.png</iconset>
   </property>
   <property name="text" >
    <string>Active jobs list columns...</string>
   </property>
  </action>
  <action name="ActionFinishedColumns_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/selectcolumns.png</iconset>
   </property>
   <property name="text" >
    <string>Finished jobs list columns...</string>
   </property>
  </action>
  <action name="ActionPreferences_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >:/resources/images/preferences.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences...</string>
   </property>
  </action>
  <action name="ActionCopyFinishedURL_" >
   <property name="text" >
    <string>Copy URL to clipboard</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContextableList</class>
   <extends>QTreeWidget</extends>
   <header>contextablelist.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
